x-pack/plugins/ml/public/application/jobs/jobs_list/components/job_details/job_details_pane.tsx

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+import { css } from '@emotion/react';
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiTitle,
+  EuiTable,
+  EuiTableBody,
+  EuiTableRow,
+  EuiTableRowCell,
+  EuiSpacer,
+  useEuiTheme,
+} from '@elastic/eui';
+
+type Item = [string, string];
+
+interface Section {
+  id: string;
+  title: string;
+  titleAction: string;
+  position: 'left' | 'right';
+  items: Item[];
+}
+
+// fix for the annotation label being hidden inside the bounds of the chart container
+
+const SectionItem: FC<{ item: Item }> = ({ item }) => {
+  return (
+    <EuiTableRow>
+      {item[0] !== '' && (
+        <EuiTableRowCell>
+          <span css={{ fontSize: '12px', fontWeight: 'bold' }}>{item[0]}</span>
+        </EuiTableRowCell>
+      )}
+      <EuiTableRowCell>
+        <span css={{ fontSize: '12px' }}>{item[1]}</span>
+      </EuiTableRowCell>
+    </EuiTableRow>
+  );
+};
+
+const Section: FC<{ section: Section }> = ({ section }) => {
+  const { euiTheme } = useEuiTheme();
+  if (section.items.length === 0) {
+    return <div />;
+  }
+
+  const cssOverride = css({
+    overflow: 'auto',
+    padding: '5px 15px',
```

Comment:
can we stick to the default EUI padding?

Reply:
Updated in https://github.com/elastic/kibana/pull/152240/commits/c0b3e32ad44dbd2646b2c2e627c3b9858d3c812b